apps/test/utilityTests.js

```diff
@@ -600,6 +600,28 @@ describe('utils.unescapeText', function () {
     assert.equal(expected, unescapeText(input));
   });
 
+  it('Adds line break for divs with attributes', function () {
+    var input, expected;
+
+    input = 'Line 1<div style="line-height: 10.8px;">Line 2</div>';
+    expected = [
+      'Line 1',
+      'Line 2'
+    ].join('\n');
+    assert.equal(expected, unescapeText(input), 'div with attribute');
+  });
+
+  it('Does not add newline for span-wrapped leading line', function () {
+    var input, expected;
+
+    input = '<span>Line1</span><div>Line2</div>';
+    expected = [
+      'Line1',
+      'Line2'
+    ].join('\n');
+    assert.equal(expected, unescapeText(input), 'first line span');
```

Comment:
Hm - should rename?  The point is that `<span>Line1</span><div>Line2</div>` maps to `Line1\nLine2` (with no _leading_ newline), not `\nLine1\nLine2` (_with_ leading newline).  We add a newline for the `<div>` but not for the `<span>`.